# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../mrxvt/no_debug_x.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://ufpr.dl.sourceforge.net/sourceforge/materm/no_debug_x.patch

--- ./src/init.c	(revision 195)
+++ ./src/init.c	(working copy)
@@ -41,7 +41,7 @@
 # define DEBUG_X
 #else
 # define DEBUG_LEVEL	0
-# define DEBUG_X
+# undef DEBUG_X
 #endif
 
 #if DEBUG_LEVEL
